packages/@misk/cli/src/utils/index.ts

```diff
@@ -33,18 +34,33 @@ export enum Files {
 
 export const JsonOptions = { spaces: 2 }
 
-export const cmdHeader = (cmd: string) =>
-  console.log(`[${cmd.toUpperCase()}] ${pwd()}`)
+export const logFormatter = (tag: string, msg?: string, dir: string = pwd()) =>
+  `[${tag.toUpperCase()}][${dir
+    .split("/")
+    .pop()
+    .toUpperCase()}] ${msg}`
 
-export const prebuild = async (...args: any) =>
-  execute("miskweb prebuild", ...args)
+export const logDebug = (tag: string, msg?: string, dir: string = pwd()) =>
+  console.log(logFormatter(tag, msg, dir))
 
-export const execute = async (cmd: string, ...args: any) => {
-  const dir = args[0]
-  if (dir) {
-    cd(dir)
+export const path = (...segments: string[]) => `${segments.join("/")}`
+
+export const parseArgs = (...args: any): { args: any[]; dir: string } => ({
+  args,
+  dir: args[0]
+})
+
+export const remove = async (path: string) => {
+  try {
+    fs.remove(path)
+  } catch (e) {
+    console.log(`[ERROR] ${e}`)
```

Comment:
again with exception tracking, why is it alright to not blow up here?

Reply:
discussed offline: `fs` library with node doesn't seem to be atomic such that we could use a `file exists` API in an if statement before calling `remove`. If the file doesn't exist, `remove` will throw an error even though it already is in the correct state. From what I've seen online, wrapping in try/catch seems to be the regularly used idiom even though it is an anti-pattern.